src/index.js

```diff
@@ -0,0 +1,287 @@
+      params: {
+        q: cityName,
+      },
+    })
+    .then((response) => {
+      const searchLocation = response.data[0];
+      console.log(response);
+      const latitudeCity = searchLocation.lat;
+      const longitudeCity = searchLocation.lon;
+      // call weather api to find temp
+      axios
+        .get('https://weather-report-sreymom-whales.herokuapp.com/weather', {
+          params: {
+            lat: latitudeCity,
+            lon: longitudeCity,
+          },
+        })
+        .then((tempResponse) => {
+          const currentTemp = Math.round(tempResponse.data.current.temp);
+
+          // get weather from weather report
+          const weatherDes = document.getElementById('des-weather');
+          const curentWeather = tempResponse.data.current.weather[0].main;
+          weatherDes.textContent = `Current Weather: ${curentWeather}`;
+
+          // get temp from weather report
+          const kelvinsTemp = document.getElementById('temp-convert');
+          kelvinsTemp.textContent = `${currentTemp}`;
+          const tempK = document.getElementById('current-temp');
+          tempK.textContent = `Temperature:   ${currentTemp}   K`;
+
+          // get humidity from weather report
+          const humidity = tempResponse.data.current.humidity;
+          const humidityDisplay = document.getElementById('humidity');
+          humidityDisplay.textContent = `Humidity:   ${humidity}`;
+
+          // get wind speed from weather report
+          const windSpeed = tempResponse.data.current.wind_speed;
+          const windSpeedDisplay = document.getElementById('wind-speed');
+          windSpeedDisplay.textContent = `Wind Speed: ${windSpeed}`;
+
+          // convert kelvin to Farenheit when click on F button
+          const convertKtoF = () => {
+            const farenheitSign = document.querySelector('.kelvin-sign');
+            const tempF = Math.round(((currentTemp - 273.15) * 9) / 5 + 3);
+            kelvinsTemp.textContent = `${tempF}`;
+            farenheitSign.textContent = '\u2109';
+            tempK.textContent = `Temperature:   ${tempF}  \u2109`;
+          };
+          const convertFarenheit = document.getElementById('farenheit');
+          convertFarenheit.addEventListener('click', convertKtoF);
+
+          // convert Kelvin to Celsius when click on C button
+          const convertKtoC = () => {
+            const celsiusSign = document.querySelector('.kelvin-sign');
+            const tempC = Math.round(currentTemp - 273.15);
+            kelvinsTemp.textContent = `${tempC}`;
+            celsiusSign.textContent = '\u2103';
+            tempK.textContent = `Temperature:   ${tempC}  \u2103`;
+          };
+          const convertCelsius = document.getElementById('degree');
+          convertCelsius.addEventListener('click', convertKtoC);
+        })
+        .catch((e) => {
+          // alert('Location does not exist!');
+          //swal('Oops!', 'Location does not exist!', 'error');
```

Comment:
Style: It's industry best practice to remove commented out code before submitting a PR.